modules/ztp-post-install-hub-factory-pipeline-checks.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="post-hub-factory-pipeline-verification-checks_{context}"]
+= Post hub factory pipeline verification checks
+
+Perform the following steps after completion of the hub factory pipeline run.
+
+.Prerequisites
+
+* An {product-title} hub cluster.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Verify ACM is successfully installed.
```

Comment:
Procedure steps should use a comma before the CLI example instead of a period.

Reply:
I added a colon